deepmd/descriptor/se_r.py

```diff
@@ -83,8 +85,6 @@ def __init__ (self,
         #        .add("activation_function", str, default = "tanh") \
         #        .add("precision",           str, default = "default")
         # class_data = args.parse(jdata)
-        if rcut < rcut_smth:
-            raise RuntimeError("rcut_smth (%f) should be no more than rcut (%f)!" % (rcut_smth, rcut))
```

Comment:
Same here

Reply:
Sorry for that. This may be because I used the previous version of code when committing mine. I will recall it.